tests/integration/it_auth_test.py

```diff
@@ -57,3 +57,12 @@ def test_forgot_password(client):
 
 def test_activate(client):
     assert client.get("/auth/activate").status_code == 405
+
+
+def test_forgot_password_post(client):
```

Comment:
@marceloamadeu The test fails

Reply:
@ammaratef45 Adjusted.

Reply:
@ammaratef45 Sorted out. I'm sorry for the large number of commits. I was having trouble with my machine's pytest. Now it's settled, too.